### PR TITLE
Update rtk_gps_hex_hereplus.md

### PR DESCRIPTION
**Pull request by hunterino**
Added instructions for pin removal, also adding image in next pull request

### CODE REVIEW

File: en/peripherals/rtk_gps_hex_hereplus.md [hunk 1]
```diff
@@ -38,3 +38,11 @@ The Here+ GPS pinout is provided below. This can be used to help modify the conn
 | 6   | BUTTON        | -   | -                 |
 | 7   | BUTTON_LED    | -   | -                 |
 | 8   | GND           | 6   | GND               |
+
+### Pin Removal instructions.
```
Comment (hamishwillee) [inline en/peripherals/rtk_gps_hex_hereplus.md]: No full stop at end of this heading

File: en/peripherals/rtk_gps_hex_hereplus.md [hunk 2]
```diff
@@ -38,3 +38,11 @@ The Here+ GPS pinout is provided below. This can be used to help modify the conn
 | 6   | BUTTON        | -   | -                 |
 | 7   | BUTTON_LED    | -   | -                 |
 | 8   | GND           | 6   | GND               |
+
+### Pin Removal instructions.  
+
+Pull the white piece of plastic between the metal stripes (I use a straight pin, to lift it up.
```
Comment (hamishwillee) [inline en/peripherals/rtk_gps_hex_hereplus.md]: Missing end bracket and new sentence not capitalised. How about replace these two lines with:

Pull the white piece of plastic between the metal stripes (e.g. use a straight pin, to lift it up) and then pull the pin out of the back side of the connector.

**Comment by hunterino** [comment]
Point well taken.  I’ll update the drawing to reflect pins 6 & 7.

**Comment by hamishwillee** [comment]
Thanks!

**Comment by hamishwillee** [comment]
Hi @hunterino , let me know when you want me to recheck!

**Comment by hamishwillee** [comment]
Ping

**Comment by hamishwillee** [comment]
Hi @hunterino , do you think you might finish this? Otherwise we should probably close it.

**Comment by hamishwillee** [comment]
I am closing this and https://github.com/PX4/px4_user_guide/pull/135 as stale. Reopen to continue work.

**Pull request closed without merging**